.cursor/rules/accessibility-requirements.mdc

```diff
@@ -0,0 +1,271 @@
+---
+alwaysApply: true
+---
+
+# Accessibility: Provide Meaningful Content for Screen Readers
+
+## Principle
+
+Ensure all UI elements provide meaningful content for screen readers and assistive technologies. Use VisuallyHidden for accessibility instead of empty strings or missing labels.
+
+## Guidelines
+
+### Do This ✅
+
+```typescript
+// Good: Provide meaningful screen reader content
+import { VisuallyHidden } from '@/components/Common'
+
+const DataTable = ({ data }) => (
+  <table>
+    <thead>
+      <tr>
+        <th>
+          Employee Name
+          <VisuallyHidden>Sort by employee name</VisuallyHidden>
+        </th>
```

Comment:
I worry a little about this particular example since the table featured in this example isn't sortable. Over adding assistive text can be a problem on its own (ex folks adding visually hidden text to buttons when the button itself should just have a better label)

Wondering if showing an icon button with aria-label might be a better example? A form field with shouldVisuallyHideLabel? Examples beyond that are a little hard to come up with for the current SDK needs